wgengine/monitor/monitor_freebsd.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package monitor
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"strings"
+)
+
+type devdConn struct {
+	conn net.Conn
+}
+
+func NewConn() (Conn, error) {
+	conn, err := net.Dial("unixpacket", "/var/run/devd.seqpacket.pipe")
```

Comment:
Does this unix socket always exist on freebsd? Or is it possible for devd to not be running on a well-behaved system?

I don't know freebsd much, so I'm trying to answer: is it reasonable to assume failure here is a serious "oh crap" error?

Reply:
To be honest, I didn't have any experience with devd before this implementation, but it seemed like the shortest path to getting it up and running on my box. I'll look into it more, but I figure it's probably better to get some semblance of "working" and iterate from there. I'm happy to drop the _freebsd part of the monitor implementation with this commit and maintain it in my own fork if there's concerns with perceptions on platform support. Let me know.

Reply:
No concerns on platform support, just trying to make sure that the freebsd port is as robust as possible. I don't know how freebsd systems are set up, so I was asking about it :)

`man devd` says:

```
The devd utility is a system daemon that runs in the background all the time.
```

So it sounds to me like that open call should not fail on a normally working system.